arches_controlled_lists/src/arches_controlled_lists/components/misc/ImportList.vue

```diff
@@ -0,0 +1,210 @@
+                detail: error.message,
+            });
+        });
+}
+
+function fileResolver() {
+    const errors = [];
+    if (!file.value) {
+        errors.push({ message: $gettext("Please select a file") });
+    } else if (!file.value.name.toLowerCase().endsWith(".xml")) {
+        errors.push({ message: $gettext("File must be an XML file") });
+    }
+    return { errors };
+}
+
+function overwriteResolver(formFieldVal: FormFieldResolverOptions) {
+    const value = formFieldVal.value;
+    const errors = [];
+    if (!value) {
+        errors.push({ message: $gettext("Please select an overwrite option") });
+    }
+    return { errors };
+}
+</script>
+
+<template>
+    <Form
+        ref="formRef"
+        class="form"
+        @submit="submit"
+    >
+        <label for="fileUpload">SKOS File</label>
+        <FormField
+            v-slot="$field"
+            name="fileUpload"
+            :resolver="fileResolver"
+            class="form-fields"
+        >
+            <InputFile
+                v-model="file"
+                accept=".xml"
+                mode="basic"
+                :auto="false"
+                :choose-label="$gettext('Choose File')"
+                :multiple="false"
+                @select="updateFileValue"
+            />
+            <Message
+                v-for="error in $field.errors"
+                :key="error.message"
+                severity="error"
+                size="small"
+            >
+                {{ error.message }}
+            </Message>
+        </FormField>
+
+        <label for="overwrite-options">Overwrite Options</label>
+        <FormField
+            v-slot="$field"
+            name="overwrite-options"
+            :resolver="overwriteResolver"
+            class="form-fields"
+        >
+            <span
+                v-for="option in overwriteOptions"
+                :key="option.value"
+                v-tooltip.bottom="{
+                    value: option.tooltip,
+                    showDelay: 1000,
+                    hideDelay: 300,
+                }"
+                class="radio-button-and-label"
+            >
+                <RadioButton
+                    v-model="overwriteOption"
+                    :input-id="option.value"
+                    :value="option.value"
+                    :initial-value="option.value"
+                    :aria-label="option.tooltip"
+                />
+                <label
+                    :for="option.value"
+                    class="radio-label"
+                    >{{ option.label }}</label
+                >
+            </span>
+            <Message
+                v-for="error in $field.errors"
+                :key="error.message"
+                severity="error"
+                size="small"
+            >
+                {{ error.message }}
+            </Message>
+        </FormField>
+        <div class="p-dialog-footer">
```

Comment:
I guess I would expect to see concepts like this using the `<template #footer>` slot rather than with the primevue classes on a div. This might mean bringing the `<Dialog>` component into this component, which I think is good anyway to declutter the outer component and not have to emit a cancel event to the parent. (This component has dialog styles anyway.)